src/@next/components/organisms/CheckoutPayment/styles.ts

```diff
@@ -0,0 +1,35 @@
+import { media, styled } from "@styles";
+
+export const Wrapper = styled.div``;
+
+export const Section = styled.section``;
+
```

Comment:
I'll fix it, but in some cases I see that it may facilitate creating new components, where you might not know in advance which html tags should be styled and which should not, so the assumption that we do all styled tags eliminates the need to change tags in JSX and move them to `styles.ts` file on a regular basis. Eventually every styled tag is rendered as single html tag, so it does not change a lot.